src/styles/card.scss

```diff
@@ -0,0 +1,77 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
```

Comment:
It's better not to use fixed values
because if the card is in a smaller container, it will better adapt

```
max-width: 200px;
width: 100%;
```